Runtime/BacktraceClient.cs

```diff
@@ -20,7 +20,7 @@ public class BacktraceClient : MonoBehaviour, IBacktraceClient
     {
         public BacktraceConfiguration Configuration;
 
-        public const string VERSION = "3.2.5";
+        public const string VERSION = "3.2.6";
```

Comment:
FWIW: We should've done 3.3.X instead following semver.org

Reply:
ah yes - but now we should continue with 3.2.x I guess?